optuna/integration/sklearn.py

```diff
@@ -243,6 +244,10 @@ def __call__(self, trial: Trial) -> float:
 
         self._store_scores(trial, scores)
 
+        test_scores = scores["test_score"]
```

Comment:
The `cross_validate()` function does not necessarily return a dict with "test_score" key, but it is only when multiple metrics are given though `scoring` parameter, which does not happen in this case.